src/tools/rbd/ArgumentTypes.h

```diff
@@ -194,6 +206,8 @@ void add_verbose_option(boost::program_options::options_description *opt);
 
 void add_no_error_option(boost::program_options::options_description *opt);
 
+void add_delay_option(boost::program_options::options_description *opt);
```

Comment:
Nit: when the argument isn't reused, I'd suggest keeping it in the associate action cc file